core/state/pruner/pruner.go

```diff
@@ -65,7 +65,9 @@ const (
 	// to avoid triggering range compaction because of small deletion.
 	rangeCompactionThreshold = 1000000
 
-	FixedPrefixAndAddrSize = 33
+	ContractTrieNodeAvgSize = 105 // This is an experience value
+
+	ContractEpochMetaAvgSize = 16 // This is an experience value
```

Comment:
Did you mean this is an **estimated** value?

Reply:
yes, it's a estimated value from previous results, it's ok for prune log now.